crypto/s2n_libcrypto.c

```diff
@@ -118,7 +118,7 @@ bool s2n_libcrypto_is_awslc()
 #endif
 }
 
-static uint64_t s2n_libcrypto_awslc_api_version(void)
+uint64_t s2n_libcrypto_awslc_api_version(void)
```

Comment:
Why remove this static?

Reply:
I need `s2n_libcrypto_awslc_api_version()` in my x509_validator_time_validation test.